.github/workflows/build.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         run: xvfb-run --auto-servernum npm run test:ci
 
       - name: "Coveralls Coverage Report"
-        uses: coverallsapp/github-action@master
+        uses: MikeEdgar/github-action@raw_coverage_file
```

Comment:
We can use official coverapps action. We are using the forked branch for affording json coveralls. We intend to switch back to the official action when this PR is merged. We do not need the fork for this repo.

Reply:
Switched to the official coveralls usage - instead of MikeEdgars.